src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -68,13 +79,13 @@
       {% set disabledField = true %}
     {% endif %}
 
-    <div class="{{ block('form_row_class') }}{% if (not compound or force_error|default(false)) and not valid %} has-error{% endif %}"{% if (attr.visible is defined and not attr.visible) %} style="display: none;"{% endif %}>
+    <div class="{{ block('form_row_class') }}{{ block('widget_type_class') }}{% if (not compound or force_error|default(false)) and not valid %} has-error{% endif %}"{% if (attr.visible is defined and not attr.visible) %} style="display: none;"{% endif %}>
```

Comment:
Move inline style to a css class (why not `d-none` ?)

Reply:
This one was already here It seems to be used by https://github.com/PrestaShop/PrestaShop/blob/33693c711738a6e6c2783cb6c01a6ae77ea25c8a/src/PrestaShopBundle/Form/Admin/Sell/Address/CustomerAddressType.php#L391 only

I agree this should be improved, but I guess it also means the javascript in address forms need to be updated as well (if we want to use d-none) It is not at all the scope of this PR so I'd suggest to open an issue to fix this in another PR

Reply:
I created an issue to fix this https://github.com/PrestaShop/PrestaShop/issues/24376